inventory/sample/group_vars/all/containerd.yml

```diff
@@ -21,6 +22,15 @@
 #     engine: ""
 #     root: ""
 
+# Example for Nvidia as additional runtime:
+# containerd_additional_runtimes:
+#   - name: nvidia
+#     type: "io.containerd.runc.v2"
+#     engine: ""
+#     root: ""
+#     options:
+#       BinaryName: "/usr/bin/nvidia-container-runtime"
+
```

Comment:
Same, this would be best in the defaults files instead.

Reply:
Move this comment example to `roles/container-engine/containerd/defaults/main.yml`